allennlp/semparse/worlds/atis_world.py

```diff
@@ -27,34 +51,72 @@ def __init__(self, utterances: List[str], tokenizer=None) -> None:
         self.utterances: List[str] = utterances
         self.tokenizer = tokenizer if tokenizer else WordTokenizer()
         self.tokenized_utterances = [self.tokenizer.tokenize(utterance) for utterance in self.utterances]
-        self.valid_actions: Dict[str, List[str]] = self.init_all_valid_actions()
+        valid_actions, linking_scores = self.init_all_valid_actions()
+        self.valid_actions: Dict[str, List[str]] = valid_actions
+
+        # This has shape (number_entities, number_utterance_tokens).
```

Comment:
`num_entities`, not `number_entities` (and same for utterance tokens).  `num_` is short for "number of", while `number_` isn't.